app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
You are comparing `masks_to_buy` to zero. This is not necessary. You could simply check `if masks_to_buy:`. This is more Pythonic and efficient.